cspell.json

```
@@ -64,19 +64,32 @@
   "words": [
     "apikey",
     "buildhome",
+    "bodycol",
     "callout",
     "checkmark",
+    "closebutton",
+    "closebtncol",
     "coachmark",
+    "coachmarks",
+    "Coachmarks",
+    "COACHMARKSTACKHOME",
```

```suggestion
    "coachmarks",
```

I thought it wasn’t case sensitive but I think if you replace `COACHMARKSTACKHOME` (I noted where I spotted it) with just `CoachmarkStackHome`, we can remove this.